src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -627,6 +627,13 @@ paths:
     post:
       operationId: ValidatorsUptime
       summary: Get Validators Uptime
+      description: |
+        Returns validators uptime data for time range limited by from_state_version and at_state_version.
+        `at_state_version` defines upper boundary (inclusive) for queried data. i.e `{ at_state_version = {epoch = 10} }`, will return uptime statistics from genesis till 10.
+        `from_state_version` defines lower boundary (inclusive) for queried data. i.e `{ from_state_version = {epoch = 10} }`, will return uptime statistics from epoch 10 till current max ledger tip.
+        it's possible to define both at once i.e `{ at_state_version = {epoch = 10}, from_state_version = {epoch = 25} }` will return uptime statistics for epochs from 10 till 25.
```

Comment:
Ideally we should be very verbose on inclusiveness and exclusiveness when documenting ranges.